test/integration/user_visits_place_test.rb

```diff
@@ -26,18 +26,35 @@ class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
     VCR.use_cassette("existing place") do
       visit "/"
 
-      page.fill_in 'nav-search', 
+      page.fill_in 'nav-search',
         :with => 'Turing School of Software & Design'
 
       click_button "Go"
 
       visit "/"
 
-      page.fill_in 'nav-search', 
+      page.fill_in 'nav-search',
         :with => 'Turing School of Software & Design'
       click_button "Go"
 
       assert_equal 1, Place.count
     end
   end
+
+  test "can comment" do
+    VCR.use_cassette("new place") do
+      login_create_place
+
+      page.fill_in 'input-comment',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.